src/rastervision/core/box_test.py

```diff
@@ -0,0 +1,155 @@
+import unittest
```

Comment:
A bit unfamiliar with how python unittest works...

Does it matter from a future-pip-installable form of rastervision that we are containing the code inside the src folders that contain the actual library code? Does it make sense to separate the test code out from the main source? (This question is possibly fueled by my jvm/scala-centric worldview)

Reply:
I don't think the current test layout would prevent making it pip installable, but if it did, it would be easy to change. You can put unit tests wherever you want. I like putting the tests alongside the source files because then you don't have to maintain a parallel file hierarchy for the tests. It's also easier to navigate to the tests for a file. This is how the Object Detection API does it. See https://github.com/tensorflow/models/tree/master/research/object_detection/core

Reply:
:+1: